lib/ansible/plugins/lookup/passwordstore.py

```diff
@@ -95,11 +100,11 @@ def parse_params(self, term):
 
     def check_pass(self):
         try:
-            self.passoutput = check_output2(["pass", self.passname]).splitlines()
+            self.passoutput = to_native(check_output2(["pass", self.passname])).splitlines()
```

Comment:
Our strategy is at the borders.  In this case, check_output is a backport of newer subprocess functionality and subprocess's API is bytes-oriented (in both py2 and py3) so it is the border.  (This is the right place to convert).  However, since this is controller code we should use to_text() to convert the output into a text string on both python2 and python3 at this border.

Note that being a string always may cause other pieces of the code to need to be turned into a text string as well.